cli/command/trust/inspect.go

```diff
@@ -0,0 +1,97 @@
+package trust
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/theupdateframework/notary/tuf/data"
+)
+
+func newInspectCommand(dockerCli command.Cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "inspect IMAGE[:TAG] [IMAGE[:TAG]...]",
+		Short: "Return low-level information about keys and signatures",
+		Args:  cli.RequiresMinArgs(1),
```

Comment:
We should either disallow specifying multiple repositories, or add the repository name to the output. The output as it is, is not useful if you specify multiple repositories;

```bash
$ docker trust inspect alpine hello-world | jq
```

Produces (removed some entries to keep it short):

```json
[
  {
    "SignedTags": [
      {
        "SignedTag": "2.6",
        "Digest": "9ace551613070689a12857d62c30ef0daa9a376107ec0fff0e34786cedb3399b",
        "Signers": [
          "Repo Admin"
        ]
      },
      {
        "SignedTag": "latest",
        "Digest": "d6bfc3baf615dc9618209a8d607ba2a8103d9c8a405b3bd8741d88b4bef36478",
        "Signers": [
          "Repo Admin"
        ]
      }
    ],
    "AdminstrativeKeys": [
      {
        "Name": "Root",
        "Keys": [
          "a2489bcac7a79aa67b19b96c4a3bf0c675ffdf00c6d2fabe1a5df1115e80adce"
        ]
      },
      {
        "Name": "Repository",
        "Keys": [
          "5a46c9aaa82ff150bb7305a2d17d0c521c2d784246807b2dc611f436a69041fd"
        ]
      }
    ]
  },
  {
    "SignedTags": [
      {
        "SignedTag": "latest",
        "Digest": "0e06ef5e1945a718b02a8c319e15bae44f47039005530bc617a5d071190ed3fc",
        "Signers": [
          "Repo Admin"
        ]
      },
      {
        "SignedTag": "linux",
        "Digest": "452733afb5319f624b88aec51006a077bccf036a87167fd657057e2e31f42736",
        "Signers": [
          "Repo Admin"
        ]
      },
    ],
    "AdminstrativeKeys": [
      {
        "Name": "Root",
        "Keys": [
          "ff544c7f4ffa0a61b2fb836fb581182181c971acd58a7023c9a3b049dc952edd"
        ]
      },
      {
        "Name": "Repository",
        "Keys": [
          "8d2aaa7461b4305f74dc80bb946ccad962829a57bd0412a480e17a2413b18ec9"
        ]
      }
    ]
  }
]
```

With that information alone, I don't know which repository the `:latest` tag belongs to

Reply:
sounds good, I added `Name` as the first field 👍